datasets/us/dod_chinese_milcorps/us_dod_chinese_milcorps.yaml

```diff
@@ -0,0 +1,30 @@
+name: us_dod_chinese_milcorps
+title: "Chinese military companies"
+url: https://www.defense.gov/News/Releases/Release/Article/3661985/dod-releases-list-of-peoples-republic-of-china-prc-military-companies-in-accord/
+summary: A list of companies identified as Chinese military companies operating in the United States.
+description: > 
+    Entities Identified as Chinese Military Companies Operating in the United States in accordance with Section 1260H of the William M. (“Mac”) Thornberry National Defense Authorization Act for Fiscal Year 2021 (Public Law 116-283).
+    Updating the Section 1260H list of "Chinese military companies" is an important continuing effort in highlighting and countering the PRC's Military-Civil Fusion strategy.
+    The PRC's Military-Civil Fusion strategy supports the modernization goals of the People's Liberation Army (PLA) by ensuring it can acquire advanced technologies and expertise developed by PRC companies, 
+    universities, and research programs that appear to be civilian entities. 
+    Section 1260H directs the Department to begin identifying, among other things, Military-Civil Fusion contributors operating directly or indirectly in the United States.
```

Comment:
```suggestion

    > Updating the Section 1260H list of "Chinese military companies" is an important continuing effort in highlighting and countering the PRC's Military-Civil Fusion strategy.
    > The PRC's Military-Civil Fusion strategy supports the modernization goals of the People's Liberation Army (PLA) by ensuring it can acquire advanced technologies and expertise developed by PRC companies, 
    universities, and research programs that appear to be civilian entities. 
    > Section 1260H directs the Department to begin identifying, among other things, Military-Civil Fusion contributors operating directly or indirectly in the United States.

```